### PR TITLE
add synonym

### LINKED ISSUE #6849: Request for new synonym [Huppke-Brendel syndrome]
### Mondo term

MONDO:0013772

### Synonyms

Huppke-Brendel syndrome

### Synonym type

exact

### Additional comments

related to https://github.com/monarch-initiative/mondo/issues/6765

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #6849

### CODE REVIEW

**Comment by twhetzel** [comment]
Converted back to Draft PR since there was an updated https://github.com/monarch-initiative/mondo/issues/6765#issuecomment-1798556243 to pause on this change until expert panel review.

**Comment by nicolevasilevsky** [comment]
I created a new ticket to add this synonym and referenced the original ticket, so we can merge this PR now and close the linked ticket.

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -274335,6 +274336,7 @@ relationship: has_characteristic MONDO:0021140 {source="https://orcid.org/0000-0
 relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/95 {source="MONDO:mim2gene_medgen"} ! SLC33A1
 property_value: confidence "2.692307692307692" xsd:double
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/4069" xsd:anyURI
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6765#issuecomment-1797800311" xsd:anyURI
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:274339]: should this then be https://github.com/monarch-initiative/mondo/issues/6849?

also, have references to a specific comment in an issue been used as the link before?

**Review by twhetzel — APPROVED** [review]
I _do_ have a question about which ticket to link this to, but otherwise fine to approve. Apologies, I thought the linked ticket was requesting both the synonym and merge be paused.

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:274339]: I think it's fine to reference the comment. I added that because it has more context.

**Pull request merged**